egs/wsj/s5/steps/nnet3/tdnn/make_configs.py

```diff
@@ -298,8 +390,11 @@ def ParseSpliceString(splice_indexes):
             'num_hidden_layers':len(splice_array)
             }
 
+# The function signature of MakeConfigs is changed just for local use in this script. 
```

Comment:
<code># The function signature of MakeConfigs is changed frequently as it is intended for local use in this script.</code>